subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -1110,7 +1110,7 @@ void zperf_shell_init(void)
 {
 	int ret;
 
-	if (IS_ENABLED(CONFIG_NET_IPV6) && MY_IP6ADDR) {
+	if (IS_ENABLED(MY_IP4ADDR_SET) && MY_IP6ADDR) {
```

Comment:
Should this be IS_ENABLED(MY_IP6ADDR_SET)

Reply:
oops, need to fix upstream as well, good catch.

Reply:
updated with the fix.